testmenu.c

```diff
@@ -69,47 +84,83 @@ int main(int argc, char **argv) {
     /* Execute SQL statement */
     rc = sqlite3_exec(db, sql, callback, 0, &zErrMsg);
 
-    if( rc != SQLITE_OK ) {
+    if( rc != SQLITE_OK )
+    {
         fprintf(stderr, "SQL error: %s\n", zErrMsg);
         sqlite3_free(zErrMsg);
     }
 
-    // Print the table
-    sql = "SELECT * FROM USERS";
+    // Print the table users for debug only
+    // sql = "SELECT * FROM USERS";
 
-    rc = sqlite3_exec(db, sql, callback, 0, &zErrMsg);
+    // rc = sqlite3_exec(db, sql, callback, 0, &zErrMsg);
 
-    if( rc != SQLITE_OK ) {
-        fprintf(stderr, "SQL error: %s\n", zErrMsg);
-        sqlite3_free(zErrMsg);
-    }
+    // if( rc != SQLITE_OK )
+    // {
+    //     fprintf(stderr, "SQL error: %s\n", zErrMsg);
+    //     sqlite3_free(zErrMsg);
+    // }
 
-    do {
+    do 
+    {
         // Case 1: User is not connected
-        if(connected == 0) {
+        if(connected == 0)
+        {
             printf("Welcome to your glycemia database, please choose an option:\n");
             printf("1. Log in with your user\n");
             printf("2. Create a new user\n");
-            printf("3. Exit\n");
+            printf("9. Exit\n");
             scanf("%d", &choice);
 
-            if(choice == 1) {
-                printf("Please enter your username:\n");
+            if(choice == 1)
+            {
+                printf("\nPlease enter your username:\n");
                 scanf("%s", &username);
+                strcat(username, "\0");
+                printf("\n");
+
                 printf("Please enter your password:\n");
```

Comment:
we still can't enter a password when creating a user
it goes to the age automatically

Reply:
shit we we see that together i've got two version, one working on my laptop and the other on the fix, we will see wich of them we keep